src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/ProjectSystem/VS/Debug/IInternalDebugLaunchProvider.cs

```diff
@@ -0,0 +1,9 @@
+﻿// Licensed to the .NET Foundation under one or more agreements. The .NET Foundation licenses this file to you under the MIT license. See the LICENSE.md file in the project root for more information.
+
+using Microsoft.VisualStudio.ProjectSystem.Debug;
+
+namespace Microsoft.VisualStudio.ProjectSystem.VS.Debug;
+internal interface IInternalDebugLaunchProvider : IDebugLaunchProvider
+{
+    public Task LaunchWithProfileAsync(DebugLaunchOptions launchOptions, ILaunchProfile profile);
```

Comment:
Access modifiers on interface members only make sense for default interface members. https://learn.microsoft.com/en-us/dotnet/csharp/language-reference/proposals/csharp-8.0/default-interface-methods#modifiers-in-interfaces

```suggestion
    Task LaunchWithProfileAsync(DebugLaunchOptions launchOptions, ILaunchProfile profile);
```